components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -91,6 +66,39 @@ function TeamUsage() {
         }
     }, [teamBillingMode]);
 
+    const loadPage = async (page: number = 1) => {
+        if (!team) {
+            return;
+        }
+        if (usagePage === undefined) {
+            setIsLoading(true);
+            setTotalCreditsUsed(0);
+        }
+        const attributionId = AttributionId.render({ kind: "team", teamId: team.id });
+        const request: ListBilledUsageRequest = {
+            attributionId,
+            fromDate: startDateOfBillMonth,
+            toDate: endDateOfBillMonth,
+            perPage: 50,
+            page,
+        };
+        try {
+            const page = await getGitpodService().server.listBilledUsage(request);
+            setUsagePage(page);
+            setTotalCreditsUsed(Math.ceil(page.totalCreditsUsed));
+        } catch (error) {
+            if (error.code === ErrorCodes.PERMISSION_DENIED) {
+                setErrorMessage("Access to usage details is restricted to team owners.");
+            }
+        } finally {
+            setIsLoading(false);
+        }
+    };
+
+    const handleSetPage = async (page: number) => {
```

Comment:
Looks like this function is not doing anything. can we just call `loadPage(page)`?